views/marts/product/fct_active_users.view.lkml

```diff
@@ -39,15 +39,30 @@ view: fct_active_users {
   dimension: daily_active_users {
     type: number
     sql: ${TABLE}.daily_active_users ;;
-    label: "DAU"
-    description: "Number of unique users for current date"
+    label: "DAU (Client)"
+    description: "Number of unique users for current date, reported by client telemetry. Equals to the number of unique users that performed an action in the given calendar date."
```

Comment:
might be worth specifying which clients are included (web, desktop, mobile, etc.)

Reply:
Left it out because it's all. Individual client data is reported in different dimensions/metrics.